### PR TITLE
add missing icon

### PR DESCRIPTION
**Pull request by utas-raymondng**

### CODE REVIEW

**Review by nspool — CHANGES_REQUESTED** [review]
Please also match the gray colour for the grid icon, and also the spacing in the original screenshot. I think the grid might need to be in it's own column, and the row height increased.

**Comment by utas-raymondng** [comment]
> Please also match the gray colour for the grid icon, and also the spacing in the original screenshot. I think the grid might need to be in it's own column, and the row height increased.

Updated as per comments

File: web/src/components/job/JobList.js [hunk 1]
```diff
@@ -90,16 +99,17 @@ const JobList = () => {
               }
             }}
           />
+          <AgGridColumn
+            width={40}
+            cellStyle={{cursor: 'pointer'}}
+            cellRenderer={(e) => (e.data?.status === 'STAGED' ? <GridOn htmlColor={'#808080'} viewBox={iconViewBoxDimension}/> : <></>)}
```
Comment (nspool) [inline web/src/components/job/JobList.js:105]: Try and avoid inline styling like this.

File: web/src/components/job/JobList.js [hunk 2]
```diff
@@ -21,6 +22,14 @@ const JobList = () => {
   const [rowData, setRowData] = useState([]);
   const [redirect, setRedirect] = useState();
   const [deleteJobId, setDeleteJobId] = useState();
+  const iconViewBoxDimension = '-2 -2 30 30';
```
Comment (nspool) [inline web/src/components/job/JobList.js:25]: Try to avoid using transforms, especially for elements in grids where there are many rows.

**Review by nspool — APPROVED** [review]

**Pull request merged**